From a29e3cc0c664b38ef311c7fc3b8d5783b84323ed Mon Sep 17 00:00:00 2001
From: louis <louis.braun@gmx.de>
Date: Sun, 31 Jan 2016 16:32:41 +0100
Subject: added svdrp command to delete image cache


diff --git a/skindesigner.c b/skindesigner.c
index c6049bb..68d94ef 100644
--- a/skindesigner.c
+++ b/skindesigner.c
@@ -198,6 +198,8 @@ const char **cPluginSkinDesigner::SVDRPHelpPages(void) {
     static const char *HelpPages[] = {
         "RELD\n"
         "    force reload of templates and caches",
+        "DLIC\n"
+        "    delete image cache",
         "SCIT\n"
         "    Set custom Integer Token key = value",
         "SCST\n"
@@ -242,6 +244,13 @@ cString cPluginSkinDesigner::SVDRPCommand(const char *Command, const char *Optio
         activeSkin->Reload();
         ReplyCode = 250;
         return "SKINDESIGNER reload of templates and caches forced.";
+    } else if (strcasecmp(Command, "DLIC") == 0) {
+        if (imgCache)
+            delete imgCache;
+        imgCache = new cImageCache();
+        imgCache->SetPathes();
+        ReplyCode = 250;
+        return "SKINDESIGNER Image Cache deleted.";
     } else if (strcasecmp(Command, "LSTF") == 0) {
         activeSkin->ListAvailableFonts();
         ReplyCode = 250;
-- 
cgit v0.10.2

