--- vdr-plugin-skindesigner-0.8.0/Makefile.orig	2016-01-31 16:06:51.818937173 +0100
+++ vdr-plugin-skindesigner-0.8.0/Makefile	2016-01-31 16:07:15.881947561 +0100
@@ -46,6 +46,7 @@
 DEFINES += -DPLUGIN_NAME_I18N='"$(PLUGIN)"' $(CONFIG)
 DEFINES += $(shell xml2-config --cflags)
 DEFINES += -DSCRIPTFOLDER='"$(SKINDESIGNER_SCRIPTDIR)"'
+DEFINES += -D__STL_CONFIG_H
 
 INCLUDES += $(shell pkg-config --cflags freetype2 fontconfig)
 INCLUDES += $(shell pkg-config --cflags librsvg-2.0 cairo-png) -ljpeg
